includes/class-give-license-handler.php

```diff
@@ -368,11 +368,20 @@ public function activate_license() {
 				return;
 			}
 
+			// Make sure license is valid.
+			// return because admin will want to activate license again.
+			if ( ! $this->is_license( $license_data ) ) {
```

Comment:
@DevinWalker I am verifying API response. In some case i will get limit access API response